django_multitenant/mixins.py

```diff
@@ -147,7 +146,20 @@ def save(self, *args, **kwargs):
 
     @property
     def tenant_field(self):
-        return self.tenant_id
+        if hasattr(self, "TenantMeta") and "tenant_field_name" in dir(self.TenantMeta):
```

Comment:
Will the new TenantMeta checks require a change in docs? If so, we might want to merge the docs PR first, then rebase this one on there and include that change.

Reply:
Yes it will require. I updated the README as well. Please check the README

Reply:
Just double-checking -- are the required changes already committed in https://github.com/citusdata/django-multitenant/pull/148 ?

Reply:
No I haven't added it. I will add after I merge the documents. Order
1. Merge the documents PR( 148)
2. Rebase with this PR
3. Add documentation changes
4. Merge this PR

Reply:
Added documentation changes and the changelog